package.json

```diff
@@ -7,8 +7,10 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "Einar Jegorov",
+  "contributors": ["Michael Rosenthal"],
   "license": "MIT",
   "dependencies": {
-    "graphile-build-pg": "^4.0.0-alpha.5"
+    "graphile-build-pg": "^4.1.0-alpha.0",
```

Comment:
This dependency _should_ be unnecessary.

Reply:
Yes, previously I had to fetch some stuff from Graphile but that is probably fixed now, these can be deleted IMO.

Reply:
Removed it, but maybe it should be in `peerDependencies`